### ISSUE TITLE
tag:master not updated in 10 months

### ISSUE THREAD

**Issue by josemariaaraujo** [hertzg/rtl_433_docker#47]
Hi,
following the documentation and needing new features added since last release, used the "master" docker image and found out it was older than the "latest"
Not sure if it is as intended, changed to "alpine-latest" and problem solved, but decided to report anyway just to made you aware of the issue (if it even is an issue at all, probably just my wrong interpretation of the documentation).

Thank you for your work in this docker image!

**Comment by hertzg** [comment]
Thanks for reporting this, it seems like this feature slipped between the cracks of #8 (happens when you have no automated tests 🙈).

I've included a fix for this in #49 could you confirm if it does once it's merged?

**Comment by hertzg** [comment]
I can see that it has tagged it correctly in the PR build logs 🎉 It will be pushed once it's merged.

https://github.com/hertzg/rtl_433_docker/runs/4519375913?check_suite_focus=true
```
--tag ***/rtl_433:master --tag ghcr.io/***/rtl_433_docker:master
```
